src/delfino/commands/test.py

```diff
@@ -28,45 +28,52 @@ def _run_tests(app_context: AppContext, name: str, maxfail: int, debug: bool) ->
 
     print_header(f"️Running {name} tests️", icon="🔎🐛")
     ensure_reports_dir(delfino)
+
+    options: ArgsList = build_args_from_dict(capture="no") if debug else []
+    args: ArgsList = [
+        "pytest",
+        "--cov",
+        delfino.sources_directory,
+        "--cov-report",
+        f"xml:{delfino.reports_directory / f'coverage-{name}.xml'}",
+        "--cov-branch",
+        "-vv",
+        *options,
+        *click_context.args,
+        delfino.tests_directory / name,
+    ]
     run(
-        list(
-            filter(
-                None,
-                [
-                    "pytest",
-                    "--cov",
-                    delfino.sources_directory,
-                    "--cov-report",
-                    f"xml:{delfino.reports_directory / f'coverage-{name}.xml'}",
-                    "--cov-branch",
-                    "-vv",
-                    "--maxfail",
-                    str(maxfail),
-                    "-s" if debug else "",
-                    delfino.tests_directory / name,
-                ],
-            )
-        ),
+        args,
         env_update={"COVERAGE_FILE": delfino.reports_directory / f"coverage-{name}.dat"},
         on_error=OnError.ABORT,
     )
 
 
-@click.command(help="Run unit tests.")
-@click.option("--maxfail", type=int, default=0)
-@click.option("--debug", is_flag=True, help="Disables capture, allowing debuggers like `pdb` to be used.")
+def test_options(func):
+    click.option(
+        "--debug",
+        is_flag=True,
+        help=(
+            "Deprecated. Use --capture=no or -s instead."
+            " | Disables capture, allowing debuggers like `pdb` to be used."
+        ),
+    )(func)
+    return func
```

Comment:
Please use the [`deprecation`](https://github.com/briancurtin/deprecation) package when you deprecate things. Example [deprecation](https://github.com/cookpad/global-python-utils/blob/b38fbccfb948b00d21ee513bc05e4280fdb1793d/src/global_python_utils/fastapi_ext/correlation_id.py#L40-L45) and [enforcing code removal](https://github.com/cookpad/global-python-utils/blob/b38fbccfb948b00d21ee513bc05e4280fdb1793d/tests/unit/fastapi_ext/test_correlation_id.py#L23).

Reply:
Not sure how should it be used for options but implemented as [7a38add](https://github.com/radeklat/delfino/pull/35/commits/7a38add04d3c2ee58825e73edf22a624edf6d7af).